Cesium3DTilesSelection/src/upsampleGltfForRasterOverlays.cpp

```diff
@@ -420,10 +424,13 @@ static bool upsamplePrimitiveForRasterOverlays(
 
   for (std::pair<const std::string, int>& attribute : primitive.attributes) {
     if (attribute.first.find("_CESIUMOVERLAY_") == 0) {
-      if (uvAccessorIndex == -1) {
-        uvAccessorIndex = attribute.second;
+      if (attribute.first.find(
+              "_CESIUMOVERLAY_" + std::to_string(textureCoordinateIndex)) ==
+          0) {
+        if (uvAccessorIndex == -1) {
```

Comment:
A few suggestions here:
- Move the construction of the attribute name outside the loop since it's loop-invariant and kinda expensive to build.
- Can you just do `attribute.first == textureCoordinateName` rather than using find? I can't see any reason why not. Should be both clearer and faster.
- Check the cheap thing (`uvAccessorIndex == -1`) before checking the expensive thing (`attribute.first == textureCoordinateName`). Won't make a huge difference, but it's a good habit.